.github/actions/env/action.yml

```diff
@@ -12,19 +12,19 @@ inputs:
     type: string
     required: true
 
-  repo_ref:
+  repo-ref:
     type: string
-  repo_ref_sha:
+  repo-ref-sha:
     type: string
-  repo_ref_name:
+  repo-ref-name:
```

Comment:
could you add some context about why these changes are necessary? Seems there are lots of replacement from `_` to `-`. But why?

Reply:
added to PR description